### ISSUE TITLE
2.0.2 Aurajoki - Call recorder

### ISSUE THREAD

**Issue by adithyakhamithkar** [dpurgin/harbour-callrecorder#53]
Hi,

I was just reading the release notes for 2.0.2 Aurajoki they say that they have a native call recorder. I wanted to know if we can still use your app on that version or there will be a conflict?

Thanks,

**Comment by dpurgin** [comment]
Hi

I haven't tested in thoroughly but the Call Recorder app works perfectly fine on 2.0.2 on my Jolla C. I've recorded a call with the native recording feature too, it worked as well.

I hope somebody can bring his/her experience too

Cheers!

**Comment by adithyakhamithkar** [comment]
Can you please help in comparison of feature of your app with the one from Jolla, I feel your app is very user friendly

**Comment by dpurgin** [comment]
Well, I guess Jolla's native feature is still in experimental phase, and my comparison is surely biased... :D But there you go:

               | harbour-callrecorder | Sailfish Feature | 
----------|---------------------|------------------------------|
Data format | FLAC (compressed) | WAV (uncomressed) | 
Minimal supported OS | Sailfish 1.1.2 | Sailfish 2.0.2 | 
Start/stop recording during the call | X | X | 
Automatic recording |  X  |  X  | 
Integration with UI (contact cards, calls, phone call UI) |   | X | 
Easy access to list of recordings | X |   | 
Search in list of recordings | X |   | 
Blacklist/whitelist feature | X |   | 
Size/age limits | X |   | 
Data relocation | X |   | 
Approval of storage after each call | X |  | 
Has bugs | X | ? |

**Comment by adithyakhamithkar** [comment]
I think your app has a advantage over the Jolla app, If we can think about adding more feature your app is no doubt going to be the first choice.

I would like to suggest if we can add a back up to onedrive/dropbox integration.

**Comment by adithyakhamithkar** [comment]
Hi,

I just moved to the Intex Aqua Fish, And I did notice that the recording happens only for SIM1 when i move my SIM to the SIM2 the recording does not happen.

Can we try debugging this?

**Comment by dpurgin** [comment]
Hi, 

it happens because the voice recording is initialized with the first modem only, that is SIM1. I'll implement the solution on my Jolla C and post the new RPM for testing

Cheers

**Comment by adithyakhamithkar** [comment]
Cool thanks I'll look forward to it.

**Comment by dpurgin** [comment]
Can you check if this build works for you? Thanks!

[harbour-callrecorder-0.7.0-2.armv7hl.zip](https://github.com/dpurgin/harbour-callrecorder/files/478664/harbour-callrecorder-0.7.0-2.armv7hl.zip)

**Comment by adithyakhamithkar** [comment]
It did work!

One more thing that i noticed was when i disabled the SIM1, I was able to record the call on SIM2, Does that make any sense to you?

**Comment by dpurgin** [comment]
SIM-cards are represented with modems /ril_0 and /ril_1, I guess once the first SIM is removed, the communication of the second SIM-card continues on /ril_0, that's why the Call Recorder is working. You can use the build I've posted in this thread, it's a working one. 

By the way, if you're moving from Jolla 1, you can use Backup/Restore feature in this latest build to move all your recordings to the new phone

**Comment by adithyakhamithkar** [comment]
Cool thanks a lot!

**Comment by adithyakhamithkar** [comment]
Hi,

After upgrading to SailfishOS 2.0.4.14 (Fiskarsinjoki)

When i try to install harbour-callrecorder-0.7.0-2.armv7hl.rpm

I am getting this error
`Fatal error: nothing provides libarchive.so.12 needed by harbour-callrecorder-0.7.0-2.armv7hl`
